### PR TITLE
website: Stage content for Dev Portal

### PR DESCRIPTION
**Pull request by topfunky**
[🔍  Preview content changes](https://developer.hashi-mktg.com/waypoint)

This PR modifies content to be rendered in the Developer Portal. This is a long lived draft PR until we are ready to launch.

Modifications include:

- Remove mention of HashiCorp Learn (reference tutorials instead)
- Fix images and code snippets to fit within the new layout
- Rearrange content and menus where necessary

### CODE REVIEW

**Comment by karl-cardenas-coding** [comment]
I validated that no documentation pages are referencing HashiCorp learn. 👍🏼

**Comment by briancain** [comment]
Is this still a valid PR @topfunky ?

**Comment by topfunky** [comment]
Yes, this is still relevant and is used by the Marketing web dev team.

**Comment by briancain** [comment]
Thank you! 😄

File: website/content/plugins/aws-ecs.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: 'Deploy and Release on AWS ECS'
 # AWS ECS
 
 -> For a step by step tutorial, visit [HashiCorp
```
Comment (tunzor) [inline website/content/plugins/aws-ecs.mdx:9]:
```suggestion
-> For a step by step guide, visit [this AWS ECS
```

File: website/content/plugins/aws-ecs.mdx [hunk 2]
```diff
@@ -7,7 +7,7 @@ description: 'Deploy and Release on AWS ECS'
...
-Learn](https://learn.hashicorp.com/tutorials/waypoint/aws-ecs?in=waypoint/docker-aws).
+Learn](https://learn.hashicorp.com/tutorials/waypoint/aws-ecs?in=waypoint/deploy-aws).
```
Comment (tunzor) [inline website/content/plugins/aws-ecs.mdx:10]:
```suggestion
tutorial](https://learn.hashicorp.com/tutorials/waypoint/aws-ecs?in=waypoint/deploy-aws).
```

**Review by tunzor — CHANGES_REQUESTED** [review]
One last mention of Learn to be removed - otherwise 👍

**Review by tunzor — APPROVED** [review]

**Pull request merged**